2017/02/09 06:35:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/09 06:47:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/09 08:30:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/09 14:06:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/09 14:15:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "media/logo_img.php".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('media/logo_img....')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/media/logo_img.php
2017/02/09 22:21:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/09 22:21:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/09 22:22:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/09 22:22:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/10 10:02:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/10 14:30:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/10 21:51:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/10 22:22:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/11 12:04:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/11 13:00:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/11 17:48:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/11 18:24:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/11 20:29:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/11 22:59:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/12 00:10:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/12 03:07:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/12 07:23:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/12 09:22:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/12 13:28:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/12 13:46:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/13 01:51:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/13 08:06:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/13 08:07:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/13 08:07:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/13 15:42:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/13 23:30:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/13 23:31:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/13 23:31:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/13 23:31:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/14 00:31:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/14 05:33:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/14 05:33:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/14 10:08:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/14 13:46:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/15 09:10:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/15 11:08:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/15 12:52:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/15 22:17:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/16 02:32:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/16 07:13:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/16 09:55:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/16 09:55:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/16 09:56:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/16 18:14:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/17 05:57:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/17 06:00:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/17 06:00:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/17 06:00:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/17 06:17:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/17 13:08:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/17 22:31:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/18 04:48:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/18 09:19:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/18 13:54:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/18 18:11:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/18 21:14:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/18 23:28:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/19 03:35:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/19 13:55:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/19 20:57:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/19 21:43:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/19 23:29:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/20 01:58:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/20 02:08:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/20 11:09:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/20 19:22:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/21 01:28:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/21 05:17:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/21 09:15:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/21 09:50:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/21 14:03:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "logs/logo_img.php".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('logs/logo_img.p...')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/logs/logo_img.php
2017/02/21 17:29:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/21 19:39:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/21 19:39:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/21 19:40:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/21 19:40:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/21 19:40:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/21 23:46:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/21 23:49:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/22 03:18:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "N0Wccvvd333.php".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('N0Wccvvd333.php')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/N0Wccvvd333.php
2017/02/22 03:18:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "N0Wccvvd333.php".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('N0Wccvvd333.php')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/N0Wccvvd333.php
2017/02/22 03:18:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "N0WaY/N0WaY1bb2/N0WaY123.php".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('N0WaY/N0WaY1bb2...')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/N0WaY/N0WaY1bb2/N0WaY123.php
2017/02/22 03:18:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "N0WaY/N0WaY1bb2/N0WaY123.php".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('N0WaY/N0WaY1bb2...')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/N0WaY/N0WaY1bb2/N0WaY123.php
2017/02/22 07:16:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/22 15:01:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/22 17:53:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/22 22:25:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
2017/02/23 02:01:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "robots.txt".' in /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php:281
Stack trace:
#0 /home/clapdev/clapd.com/yii/framework/web/CWebApplication.php(136): CWebApplication->runController('robots.txt')
#1 /home/clapdev/clapd.com/yii/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /home/clapdev/clapd.com/index.php(15): CApplication->run()
#3 {main} REQUEST_URI=/robots.txt
